src/deploy.js

```diff
@@ -54,7 +112,7 @@ const deploy = async (opts) => {
   const client = s3.createClient({ s3Client: new AWS.S3() });
 
   const params = {
-    localDir: BUILD,
+    localDir: build,
```

Comment:
Did not see comment on left side for the delete removed comments:

> Put both lines of comment above deleteRemoved.

Reply:
@juancarlosfarah Please let me know if here is still something pending